src/App.css

```diff
@@ -1,3 +1,40 @@
-.App {
-  text-align: center;
+@import url("https://saraniui.netlify.app/common_css/sarani_ui.css");
+
+/*Reset CSS*/
+* {
+  margin: 0;
+  font-family: "Inter", sans-serif;
+  box-sizing: border-box;
+  color: var(--text-color);
+  line-height: 1.5;
+}
+
+:root {
+  --text-color: #333;
+  --text-color-light: #5a5a5a;
+  --text-color-white: #eee;
+  --pure-white-color: #fff;
+
+  --primary-color: #00aaff;
+  --primary-color-hover: #0088cc;
+
+  --shadow-color: gray;
+  --shadow-color-light: rgb(0 0 0 / 10%);
+  --light-gray-bg: #eaeaec;
+
+  --cta-color: #fb641b;
+  --cta-color-hover: #ff8200;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+a {
```

Comment:
We will use Link from react router so no need to add any styles for `a`

Reply:
Thank You Jantu. I removed this.